src/components/DpBadge/DpBadge.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
Please bootstrap minimal documentation:

- DpBadge.stories.mdx
- DpBadge.stories.tsx

Several simple examples can be found in existing components. I'll be happy to assist with further documentation content.

Reply:
We decided it should be better if the recommendation modal will be refactored to a generic modal which is able to load both addons. This way the addon warpper can remain in the core. https://github.com/demos-europe/demosplan-ui/pull/473/commits/9e235a911cbe304196bd256283c3f63d45a85c3f but with the changes you mentioned in this PR

Reply:
But DpBadge can very well live in demosplan-ui!

Reply:
Sorry my bad I was just refering to the addon wrapper- Added the bootstrapped stories for the DpBadge component here:
https://github.com/demos-europe/demosplan-ui/pull/473/commits/d93a3fa6216d242bc6dd5950d59500959e7667ee